### ISSUE TITLE
Labels from Inception at the beginning of training seem incorrect

### ISSUE THREAD

**Issue by siggi90** [baldassarreFe/deep-koalarization#21]
When training the network and looking at the outputted images from the validation set, i see labels from inception at the top of the images, are they supposed to be objects recognized in the images from the pretrained inception features? If so they seem very incorrect, i see objects listed which are not present in the images and vice versa. Is this because the network is untrained and doesn't utilize the features from inception correctly? Also how long does it usually take for the network (on say 10.000 images, i have a larger dataset but I've started of with a small portion to test it out) to converge to some colors being generated because after about 200 iterations all i see is a redish-brownish color on all the images. Is it possible I'm doing something wrong?
